Azure-Samples/azure-iot-samples-csharp
# refreshed simulator.exe

## Purpose

* ...
On clean installs of Windows, Simulator.exe was failing with a message that updating it might fix the problem. Refreshing it did fix the issue. Raised in GH issue 36775
## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
Refreshed version of Simulator.exe

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x ] Other... Please describe: GH issue fix
```

## How to Test
*  Get the code
Follow the steps in this topic: https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-tpm-csharp  on a FRESH install of Windows (I used Windows 10 v 1809). The sample should run smoothly with no error window from Simulator.exe.
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
